app/test_main.py

```diff
@@ -1,3 +1,35 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (10, 10, [0, 0]),
+        (15, 15, [1, 1]),
+        (20, 20, [1, 1]),
+        (24, 24, [2, 2]),
+        (28, 28, [3, 2]),
+        (40, 40, [6, 5]),
+        (16, 40, [1, 5]),
+        (40, 16, [6, 1]),
+        (0, 0, [0, 0]),
+        (100, 100, [21, 17]),
```

Comment:
Ensure that the test case with maximum years (100, 100) is valid and that the expected output [21, 17] is correct. This test should check if the function handles large input values properly, as mentioned in the checklist.